java/RSPEC-128.java

```diff
@@ -0,0 +1,14 @@
+class SwitchNoBreak {
+    void test() {
+        switch (myVariable) {
+            case 1:
+                foo();
+                break;
+            case 2:  // Both 'doSomething()' and 'doSomethingElse()' will be executed. Is it on purpose ?
```

Comment:
RSPEC-128: Switch without break
<!-- lookout footnote separator -->
_Comment made by the analyzer sonarcheck_analyzer._